PokerChallenge/PokerChallenge(2)/PokerHandTypes/Flush.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace PokerChallenge2
+{
+    public static class Flush
+    {
+        public static bool IsFlush(this PokerHand hand)
```

Comment:
I really like this use of extension methods. Isolate of code in the flush class makes similar code in one area while giving functionality to all that reference poker hand. This is like a perfect example of how to use them.